src/components/registration/steps.jsx

```diff
@@ -455,7 +455,11 @@ module.exports = {
             };
         },
         onValidSubmit: function (formData, reset, invalidate) {
-            if (!formData.phone || formData.phone.national_number === '+') {
+            if (!formData.phone) {
```

Comment:
From what @jwzimmer mentioned in the issue, I think that just having the `+` is also a situation in which we'd want to show the `validationRequired` message. I'd recommend keeping the logic statement as it was before, but just changing the message from `validationPhoneNumber` to `validationRequired`, if that makes sense.